docs/content/en/docs-dev/operator-manual/piped/installation/installing-on-cloudrun.md

```diff
@@ -68,6 +68,8 @@ description: >
   gcloud secrets create cloudrun-piped-config --data-file={PATH_TO_CONFIG_FILE}
   ```
 
+  then make sure that Cloud Run has the ability to access that secret as [this guide](https://cloud.google.com/run/docs/configuring/secrets#access-secret).
```

Comment:
nits, should it be `CloudRun` instead of `Cloud Run` 😄

Reply:
Actually, the official name is `Cloud Run` so we have to fix our docs later.